core/config/yaml.go

```diff
@@ -117,12 +119,23 @@ func (y yamlConfigProvider) Name() string {
 
 // GetValue returns a configuration value by name
 func (y yamlConfigProvider) GetValue(key string) Value {
-	node := y.getNode(key)
+	// check the cache for the value
+	//
```

Comment:
remove this empty line, this is ugly and not idiomatic (even if it already exists in this file)